ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
 
-The P-Chain can produce a `SubnetConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
+The P-Chain can produce a `SubnetToL1ConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
 
-The following serialization is defined as a `ValidatorData`:
+The following serialization is defined as the `ValidatorData`:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `nodeID` | `[]byte` | 4 + len(`nodeID`) bytes |
-| `blsPublicKey` | `[48]byte` | 48 bytes |
-| `weight` | `uint64` | 8 bytes |
-| | | 60 + len(`nodeID`) bytes |
+|          Field |       Type |                     Size |
+| -------------: | ---------: | -----------------------: |
+|       `nodeID` |   `[]byte` |  4 + len(`nodeID`) bytes |
+| `blsPublicKey` | `[48]byte` |                 48 bytes |
+|       `weight` |   `uint64` |                  8 bytes |
+|                |            | 60 + len(`nodeID`) bytes |
 
+The following serialization is defined as the `ConversionData`:
 
-The following serialization is defined as the `SubnetConversionData`:
-
-| Field | Type | Size |
-| -: | -: | -: |
-| `codecID`  | `uint16` | 2 bytes |
-| `subnetID` | `[32]byte` | 32 bytes |
-| `managerChainID` | `[32]byte` | 32 bytes |
-| `managerAddress` | `[]byte` | 4 + len(`managerAddress`) bytes |
-| `validators` | `[]ValidatorData` | 4 + sum(`validatorLengths`) bytes |
-| | | 74 + len(`managerAddress`) + len(`validatorLengths`) bytes |
+|            Field |              Type |                                                       Size |
+| ---------------: | ----------------: | ---------------------------------------------------------: |
+|        `codecID` |          `uint16` |                                                    2 bytes |
+|       `subnetID` |        `[32]byte` |                                                   32 bytes |
+| `managerChainID` |        `[32]byte` |                                                   32 bytes |
+| `managerAddress` |          `[]byte` |                            4 + len(`managerAddress`) bytes |
+|     `validators` | `[]ValidatorData` |                          4 + sum(`validatorLengths`) bytes |
+|                  |                   | 74 + len(`managerAddress`) + sum(`validatorLengths`) bytes |
 
 - `codecID` is the codec version used to serialize the payload, and is hardcoded to `0x0000`
 - `sum(validatorLengths)` is the sum of the lengths of `ValidatorData` serializations included in `validators`.
-- `subnetID` identifies the Subnet that is being converted (described further below).
-- `managerChainID` and `managerAddress` identify the validator manager for the given Subnet. This is the (blockchain ID, adress) tuple allowed to send Warp messages to modify the Subnet's validator set.
-- `validators` are the initial pay-as-you-go validators for the given Subnet.
+- `subnetID` identifies the Subnet that is being converted to an L1 (described further below).
```

Comment:
```suggestion
- `subnetID` identifies the Subnet that is being converted to a L1 (described further below).
```

Reply:
> these indefinite articles are used [according to the sound](https://www.merriam-webster.com/grammar/is-it-a-or-an) that starts the following word, not the letter. So, we say “a dog” and “a balloon” or “an urgent message” and “an ant” but also “a useless pen” and “an hour.”

"L" said _out loud_ begins with a vowel sound; therefore the appropriate article is "an".
https://www.pristineword.com/a-an/#tit05